topic5/src/main/java/topic5/userService/UserServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package topic5.userService;
+
+import java.util.LinkedList;
+
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import topic5.UserNotFoundException;
+
+@RestController
+public class UserServiceImpl implements UserService {
+
+	private LinkedList<User> users = new LinkedList<User>();
```

Comment:
Instead of List why not use a Map?

Reply:
You are right, i was going to use a HashMap initially, i don't know why i changed it.

I'll implement it, the HashMap is better suited for the task.